src/plugins/discover/public/components/discover_grid/discover_grid.tsx

```diff
@@ -93,6 +94,10 @@ export interface DiscoverGridProps {
    * Determines if data is currently loaded
    */
   isLoading: boolean;
+  /**
+   * Determines if it's loading more data (next chunk)
+   */
+  isLoadingMore?: boolean;
```

Comment:
Not a request to change anything here, but a thought on data structure design :thinking: since it can't be `loading` and `loading_more` at the same time, a single field which indicates either `loading`, `loading_more` or `idle` could represent this in a way that makes invalid state unrepresentable.